src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,75 @@
+package lotto.controller;
+
+import lotto.domain.BonusNumber;
+import lotto.domain.LottoResult;
+import lotto.domain.Lottos;
+import lotto.domain.Money;
+import lotto.domain.PickedNumbers;
+import lotto.domain.WinningLotto;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+    private InputView inputView;
+    private OutputView outputView;
+    private Lottos lottos;
+
+    public LottoController() {
+        this.inputView = new InputView();
+        this.outputView = new OutputView();
+    }
+
+    public void start() {
+        Money money = getMoney();
+        purchaseLotto(money);
+        WinningLotto winningLotto = getWinningLotto();
+        outputView.printResult(lottos.getResult(winningLotto));
+        outputView.printYield(lottos.getYield(money));
+        inputView.terminate();
+    }
+
+    private WinningLotto getWinningLotto() {
+        PickedNumbers pickedNumbers = getPickedNumber();
+        BonusNumber bonusNumber = getBonusNumber(pickedNumbers);
+        WinningLotto winningLotto = new WinningLotto(pickedNumbers, bonusNumber);
+        return winningLotto;
```

Comment:
이런 경우에는 불필요한 변수를 생성하지 않고 바로 반환해도 괜찮다고 생각해요. 

왜냐하면 생성자기에 읽는이가 충분히 `winningLotto`를 생성한다는 내용을 유추할 수 있거든요!

다만 판다가 의미를 부여하고 싶을 경우에는 변수로 빼는 것도 좋은 선택이라 보여집니다!

```suggestion
        return  new WinningLotto(pickedNumbers, bonusNumber);
```

Reply:
이 부분은 처음엔 토니가 말해주신 방식대로 작성했었는데, 다른 사람이 보았을 때 가독성이 떨어지는 것이 아닌가 하는 우려로 페어인 아서와 의논해 따로 분리했었던 부분이네요. 
토니가 보기에 합쳤을 때 가독성 문제가 없는 것 같다면 다시 바로 반환하는 쪽으로 작성할게요!

Reply:
페어와 함께 상의하여 결론내셨군요~ 
저는 이런 상황에서 가독성 문제가 없다고 생각하여 불필요한 변수를 선언하지 않는걸 선호하지만 판다가 원하는대로 구현하셔도 될거 같아요!